README.md

```diff
@@ -1,30 +1,49 @@
 
-- Clear steps for instrumenting your services
+- Clear instructions for instrumenting your services
 - Observability building blocks like dashboards and alerts
 
 All of this is available through an open ecosystem where New Relic developers, partners, and customers contribute their best-practice solutions.
 
+## Quick start components
+
+### 📊 Dashboards
+
+With [New Relic One dashboards](https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) you can customize and understand the data you collect. Explore your data and correlate connected sources with tailored, user-friendly charts, and quickly learn the state of your system and applications for faster, more efficient troubleshooting.
+
+> You can't copy/paste a dashboard's json directly from the repository to create a dashboard within New Relic. You will need to add "permissions": "PUBLIC_READ_WRITE",to the json prior to adding the dashboard to your account.
+
+### 📟 Alerts
+
+[Alerts](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/introduction-alerts/) lets you set up robust and customizable alert policies for anything that you can monitor. Receive notifications for fluctuations in key performance metrics as data streams in from all of our products, including APM, infrastructure, browser, mobile, and NRQL queries.
+
+### 💽 Data Sources
+
+[Data Sources](https://developer.newrelic.com/collect-data/collect-data-from-any-source/) are the many sources of data you can setup with New Relic to collect and monitor data.
+
+### 🔮 Future
+
+Our road map for quick starts includes the addition of [Synthetics monitoring](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring/), [NerdPacks](https://developer.newrelic.com/build-apps/) and [Flex configurations](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration/).
+
 ## Getting Started
 
-Contribute your own observability pack to the New Relic One catalog by following the steps below:
+Contribute your own quick start to the New Relic One catalog by following the steps below:
 
 1. [Fork the Github repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository)
 
 2. [Clone your own repository to your local machine](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
 
-3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your pack, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`
+3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your Quick start, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`
```

Comment:
just changing to small case to keep it consistent
```suggestion
3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your quick start, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`
```

Reply:
```suggestion
3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your quickstart, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`
```